specification/communication/resource-manager/Microsoft.Communication/stable/2020-08-20/CommunicationService.json

```diff
@@ -670,6 +670,10 @@
           "description": "Name of the operation with format: {provider}/{resource}/{operation}",
           "type": "string"
         },
+        "isDataAction": {
```

Comment:
Can you add new properties to the end of the list? Otherwise the ctor signature may be breaking change.